microsoft-edge/web-platform/os-regional-settings.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Updates to Region information exposed to web content
+description: How customers and web developers can use the OS regional format in Microsoft Edge for improved site experiences
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 08/11/2022
+--- 
+# Updates to Region information exposed to web content
+
+Microsoft Edge provides operating sysytem (OS) regional preference information to web content to help web site authors create regional experiences on their website. This feature allows website authors to deliver value to customers who specifically change their regional preferences in the OS to reflect their personal preferences.
+
+Users want to see dates, times, numbers, and other formatting printed according to their preferences. For example, the US geographic region translates **7/2/2022** into **July 2nd, 2022**.  Another geographic region could translate that same date into **February 7, 2022** because they read the month and day inversely. 
+
+## Regional Codes
+The user’s formatting preferences are abstracted into a simple regional code, which is used by web applications to infer the various details noted above about dates, times, and other formatting preferences. With a regional code, websites can use a built-in Internationalization API to properly format information into the user's preferences. The built-in Internationalization API saves lots of MB of data transfer over the network, keeping websites small and allowing them to quickly and easily adapt to different regions. In general, the website's internationalization customer experiences are only as good as the regional code that they're able to determine for the current user. 
+
+A regional code is a two-letter language code followed by a hyphen and a two-letter region code.  Microsoft Edge uses ICU, which is an implementation of the Unicode standard's Internationalization API.  See [ICU](https://icu.unicode.org/home).
+
+To set the user’s language preferences, Microsoft Edge gets the default values from the OS at the time it's first installed. After that, users can change the language preferences on **edge://settings/languages** and then sorting them in priority order.
+
+|Microsoft Edge Setting Category| Description |
+|--------|-------------------------|
+| The language used for rendering Microsoft Edge's UI | edge://settings/languages. One item in the list will indicate **Microsoft Edge is displayed in this language**|
+| The Language used to render web sites | **Website Server-side processing**: Microsoft Edge sends the language list (as regional codes) specified in **edge://settings/languages** via the accept-language header. <br><br> **Website Client-side processing**: Microsoft Edge makes the language list (as regional codes) available via the `navigator.languages` DOM API. The preferred language (at the top of the list) is also available as `navigator.language`. |
```

Comment:
```suggestion
| The Language used to render web sites | **Website Server-side processing**: Microsoft Edge sends the language list (as regional codes) specified in **edge://settings/languages** via the `accept-language` header. <br><br> **Website Client-side processing**: Microsoft Edge makes the language list (as regional codes) available via the `navigator.languages` DOM API. The preferred language (at the top of the list) is also available as `navigator.language`. |
```

Reply:
"preferred language" is not-bold on this row, yet "displayed language" is bold on next row.  consistent?